Documentation/network/egress-gateway/egress-gateway.rst

```diff
@@ -268,14 +270,25 @@ There are 3 different ways this can be achieved:
        nodeSelector:
          matchLabels:
            testLabel: testVal
-       egressIP: a.b.c.d
+       egressIP: a.b.c.d  # IPv4 example
+
+   For IPv6, you can specify an IPv6 address:
+
+   .. code-block:: yaml
+
+     egressGateway:
+       nodeSelector:
+         matchLabels:
+           testLabel: testVal
+       egressIP: 2001:db8::1  # IPv6 example
```

Comment:
I don't think we need to instruct users how to specify an IPv6 address. The text above declares that IPv6 is supported, and if the user wants to use IPv6 then they will know how an IPv6 address looks. It doesn't look like there's any special required syntax like wrapping it in speech marks or anything, so we can just keep the existing example.

```suggestion
       egressIP: a.b.c.d
```